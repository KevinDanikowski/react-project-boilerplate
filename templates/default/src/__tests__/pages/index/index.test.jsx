import React from 'react'
import renderer from 'react-test-renderer'
import { useQuery } from '@apollo/react-hooks'
import App from '../../../pages/index'

jest.mock('@apollo/react-hooks')
useQuery.mockReturnValue({loading: true})

describe('Pages:Homepage', () => {
  it('renders the homepage', () => {
    const component = renderer.create(<App />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
