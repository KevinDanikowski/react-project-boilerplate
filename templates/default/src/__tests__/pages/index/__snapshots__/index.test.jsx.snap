// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pages:Homepage renders the homepage 1`] = `
<div
  style={
    Object {
      "textAlign": "center",
    }
  }
>
  <p>
    Welcome to the boilerplate React app using NextJS!
  </p>
  <p>
    Try navigating to
    <a
      href="/example/second-page"
      onClick={[Function]}
      onMouseEnter={[Function]}
    >
       another page 
    </a>
    and observe the URL changing
  </p>
  <p>
    Or try interacting with our graphql country endpoint
  </p>
  <div>
    Loading Countries...
  </div>
</div>
`;
